### PR TITLE
Add discoverer component name 

### PR DESCRIPTION
**Pull request by anatheka**

### CODE REVIEW

File: service/discovery/azure/azure.go [hunk 1]
```diff
@@ -42,6 +42,12 @@ import (
 	"clouditor.io/clouditor/internal/util"
 )
 
+const (
+	StorageComponent = "storage"
+	ComputeComponent = "compute"
+	NetworkComponent = "network"
+)
```
Comment (lebogg) [inline service/discovery/azure/azure.go:49]: Maybe we move these one level up s.t. all discoveries use the same constants, e.g. also AWS. What do you think?

**Review by lebogg — COMMENTED** [review]
LGTM. Only one minor idea.

File: service/discovery/azure/azure.go (hunk 1 above)
Comment (anatheka) [reply service/discovery/azure/azure.go:49]: You are absolutely right. I will change that.

Comment (anatheka) [reply service/discovery/azure/azure.go:49]: If we change this we run into 'import cycle' errors. For now, I'll leave it like that.

**Review by lebogg — APPROVED** [review]

**Pull request merged**